### PR TITLE
Add script to copy data for manual data request

### PR DESCRIPTION
**Pull request by andylangridge**
* Adds a script to initiate the data retrieval process for a manual audit data request
* Since there was also a sendResult script, I moved this code into a shared folder `scripts/manualAuditDataRequests` along with some code for the new script (separated into sub-folders)

### CODE REVIEW

File: scripts/manualAuditDataRequestInitiateCopyAndDecrypt.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    Math.floor(Math.random() * 1000000).toString()
+  )
+
+program.parse(process.argv)
+
+const options = program.opts()
+const dates: string[] = options.dates
+const zendeskId: string = options.zendeskId
+
+if ((!dates && !isStringArray(dates)) || !zendeskId) {
```
[the remark this answers is not in the document]
Comment (andylangridge) [reply scripts/manualAuditDataRequestInitiateCopyAndDecrypt.ts:23]: Good point, shall look at adding!

[the remark this answers is not in the document]
Comment (andylangridge) [reply scripts/manualAuditDataRequestInitiateCopyAndDecrypt.ts]: I updated the `isStringArray` function to `isDatesArray` and added some tests for it. Let me know what you think :)

File: scripts/manualAuditDataRequests/initiateCopyAndDecrypt/isDatesArray.test.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { isDatesArray } from './isDatesArray'
+
+describe('isDatesArray', () => {
+  it('should return true if the array is an array of dates', () => {
+    const testArray = ['2020-01-01', '2020-01-02']
```
Comment (gdsandrewmccafferty) [inline scripts/manualAuditDataRequests/initiateCopyAndDecrypt/isDatesArray.test.ts:5]: Minor point, but you could inline all these variables, there's no need to declare something you're only using once, unless it adds clarity, which in this case I don't think it does.

Comment (andylangridge) [reply scripts/manualAuditDataRequests/initiateCopyAndDecrypt/isDatesArray.test.ts]: Done

File: scripts/sharedServices/aws/sqs/sendSqsMessage.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import {
```
Comment (gdsandrewmccafferty) [inline scripts/sharedServices/aws/sqs/sendSqsMessage.ts:1]: As discussed, let's not copy and paste something that's nearly identical here, and just set an environment variable in code to let us use the version of the same thing in `src`

**Pull request merged**